src/axom/core/Map.hpp

```diff
@@ -778,15 +778,23 @@ class Map
    * \param [in] index the index needed for locking, if this weren't sequential.
    * \param [in] overload execution space object for the sake of function overloading.
    */
-  void bucket_lock(std::size_t index, axom::SEQ_EXEC overload) const { (void)index; (void) overload; } 
+  void bucket_lock(std::size_t index, axom::SEQ_EXEC overload) const
+  {
+    (void)index;
+    (void)overload;
```

Comment:
Please change these to:
```suggestion
    AXOM_UNUSED_VAR(index);
    AXOM_UNUSED_VAR(overload);
```